crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -356,6 +356,23 @@ impl Room {
         self.inner.read().unwrap().latest_event.clone()
     }
 
+    /// Return the most recent few encrypted events. When the keys come through
+    /// to decrypt these, the most recent relevant one will replace
+    /// latest_event. (We can't tell which one is relevant until
+    /// they are decrypted.)
+    pub fn latest_encrypted_events(&self) -> RingBuffer<Raw<AnySyncTimelineEvent>> {
```

Comment:
```diff
- pub fn
+ pub(crate) fn
```

or even:

```diff
- pub fn
+ fn
```

I would not make this method available everywhere :-). The `RingBuffer` is also an inner implementation details.